mlflow/tracking/fluent.py

```diff
@@ -286,6 +303,34 @@ def log_artifacts(local_dir, artifact_path=None):
     MlflowClient().log_artifacts(run_id, local_dir, artifact_path)
 
 
+def list_run_infos(experiment_id, run_view_type=ViewType.ACTIVE_ONLY):
```

Comment:
I think we might want to hold off on exposing this method via the fluent API - it's subsumed by the fluent `search_runs` API, which takes care to handle the fact that the underlying `SearchRuns` REST API is paginated & avoid OOMing the caller's machine in the common case. Let me know what you think - IMO we should see if there's a promising use case for listing run infos & expose this API then, rather than in this PR.